toolbox/deps_update/githubClient.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2017 Istio Authors
```

Comment:
I actually see a mix of camel case and snake case in our repos for go files and got confused too. But I guess I will leave it like this unless people have a strong opinion